packages/eslint-plugin/docs/rules/no-deprecated.mdx

```diff
@@ -59,6 +59,57 @@ const url2 = new URL('/foo', 'http://www.example.com');
 </TabItem>
 </Tabs>
 
+## Options
+
+### `allow`
+
+{/* insert option description */}
+
+This option takes the shared [`TypeOrValueSpecifier` format](/packages/type-utils/type-or-value-specifier).
+
+Examples of code for this rule with:
+
+```json
+{
+  "allow": [
+    { "from": "file", "name": "apiV1" },
+    { "from": "lib", "name": "escape" },
+    { "from": "package", "name": "Bar", "package": "bar-lib" }
```

Comment:
Nit: since the package isn't referenced in the examples..

```suggestion
```

Reply:
Oh. Thanks!
[4ee9616](https://github.com/typescript-eslint/typescript-eslint/pull/10585/commits/4ee961679dfcf8da8cbe395267bd9bc88eb6ee3c)